src/patternfly/components/AboutModalBox/examples/about-modal-box-example.hbs

```diff
@@ -4,7 +4,7 @@
     {{/about-modal-box-brand-image}}
   {{/about-modal-box-brand}}
   {{#> about-modal-box-close about-modal-box-close--attribute='aria-label="Close Dialog"'}}
-    {{#> button button--modifier="pf-m-plain"}}
+    {{#> button button--attribute='aria-label="Close Dialog"' button--modifier="pf-m-plain"}}
```

Comment:
I missed this in my original review. Can you make a couple of updates here:

- Currently, the element `div.pf-c-about-modal-box__close` includes `aria-label="Close Dialog"`, which isn't needed. I think you can remove it by removing `about-modal-box-close--attribute='aria-label="Close Dialog"'` from `about-modal-box-close`
- Our [capitalization guidelines](https://v2.patternfly.org/design-guidelines/content/grammar-and-terminology#capitalization) for PF4 uses sentence style capitalization. Can you change "Close Dialog" on the button to "Close dialog"

Reply:
Thanks for pointing this out @jgiardino! Corrections has been added.